imports/plugins/included/product-variant/client/templates/products/productDetail/variants/variantForm/variantForm.js

```diff
@@ -167,7 +181,12 @@ Template.variantForm.events({
     }
     Meteor.call("products/cloneVariant", productId, template.data._id,
       function (error, result) {
-        if (result) {
+        if (error) {
+          Alerts.alert({
+            text: i18next.t("productDetailEdit.cloneVariantFail", { title: template.data.title }),
```

Comment:
I suspect that `  "productDetailEdit": {` could all be moved into package `server/i18n`.